cmd2/cmd2.py

```diff
@@ -2268,7 +2268,7 @@ def alias_list(self, args: argparse.Namespace):
     alias_parser = ACArgumentParser(description=alias_description, epilog=alias_epilog, prog='alias')
 
     # Add subcommands to alias
-    alias_subparsers = alias_parser.add_subparsers()
+    alias_subparsers = alias_parser.add_subparsers(title='sub-commands')
```

Comment:
Thanks for this change.  "sub-commands" is a much better title for the group of subcommands than "positional arguments" or whatnot ;-)